mailpoet/tests/integration/Newsletter/Scheduler/PostNotificationTest.php

```diff
@@ -63,22 +71,26 @@ public function testItCreatesPostNotificationSendingTask() {
     ]);
 
     // new queue record should be created
-    $queue = $this->postNotificationScheduler->createPostNotificationSendingTask($newsletter);
-    assert($queue instanceof SendingTask);
-    expect(SendingQueue::where('newsletter_id', $newsletter->getId())->findMany())->count(1);
-    expect($queue->newsletterId)->equals($newsletter->getId());
-    expect($queue->status)->equals(SendingQueue::STATUS_SCHEDULED);
+    $sendingQueue = $this->postNotificationScheduler->createPostNotificationSendingTask($newsletter);
+    assert($sendingQueue instanceof SendingQueueEntity);
+    assert($sendingQueue->getTask() instanceof ScheduledTaskEntity);
+    expect($this->sendingQueuesRepository->findBy(['newsletter' => $newsletter]))->count(1);
+    if ($sendingQueue->getNewsletter())
+      expect($sendingQueue->getNewsletter()->getId())->equals($newsletter->getId());
```

Comment:
I think that this check shouldn't be behind a condition. Please, don't forget to check the rest of your changes.